backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -680,6 +682,15 @@ object VeloxConfig extends ConfigRegistry {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("10MB")
 
+  val ENABLE_LIMIT_BATCH_RESIZER = {
```

Comment:
This can be a default behavior, please remove this config, all the operator should respect `preferredBatchBytes`